ray_on_golem/server/main.py

```diff
@@ -52,7 +52,7 @@ def main(port: int, self_shutdown: bool, registry_stats: bool):
         web.run_app(
             app,
             port=app["port"],
-            print=None,
+            print=logger.info,
```

Comment:
How much does it log? Maybe `debug` would be better here?

Reply:
hmm.... if it's something that gets _printed_ normally, I don't think there's much... and I do believe what _is_ being output would constitute regular info

Reply:
ok